### PR TITLE
fix(beta): add privilege for feature preview

### PR DESCRIPTION
**Pull request by petrjasek**

### CODE REVIEW

File: client/app/scripts/superdesk/menu/menu.js [hunk 1]
```diff
@@ -88,6 +100,11 @@
                     });
 
                     scope.notifications = userNotifications;
+
+                    privileges.loaded.then(function() {
+                        console.log('in', privileges.privileges);
```
Comment (mugurrus) [inline client/app/scripts/superdesk/menu/menu.js]: console.log :)

**Comment by mugurrus** [comment]
:+1:

File: client/app/scripts/superdesk/menu/menu.js (hunk 1 above)
Comment (petrjasek) [inline client/app/scripts/superdesk/menu/menu.js]: thx :)

**Comment by sivakuna-aap** [comment]
:+1:

**Comment by ioanpocol** [comment]
:+1:

**Pull request merged**